packer/install-core-packages.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -eux
```

Comment:
We should discuss making all those install scripts used by both packer and kubernetescustomscript for CSE installs to avoid having duplicated code and different install approaches + be able to change/tune install code in one place as the code evolves. Also would make sure that clusters with and without golden images get the exact same stuff.